source/core_http_client.c

```diff
@@ -916,7 +898,7 @@ static int httpParserOnHeadersCompleteCallback( llhttp_t * pHttpParser )
      * parsing here. */
     if( ( pResponse->respOptionFlags & HTTP_RESPONSE_DO_NOT_PARSE_BODY_FLAG ) != 0U )
     {
-        shouldContinueParse = LLHTTP_PAUSE_PARSING;
+        shouldContinueParse = ( int ) LLHTTP_PAUSE_PARSING;
```

Comment:
Is there a reason this variable doesn't follow the naming convention?

Reply:
I am not sure I understand. I think `shouldContinueParse` does follow naming convention - looking [here](https://github.com/FreeRTOS/coreHTTP/blob/65eba45d38614f62e3a1e178c6fd956fb90a1e34/source/core_http_client.c#L2370) for reference.

Reply:
Seems more like there are multiple places that this occurs. Fixing this is outside the scope of this PR though, so resolving this conversation.